### ISSUE TITLE
NTR: Hemimegalencephaly

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4061]
Synonyms: NA 
; HPO ID: NA 
; Alternative ID: Cortical dysplasia 
; Son of_1: NA 
; Son of_2: NA 
; Son of_3: NA 
; Son of_4: A malformation of cortical development characterized by enlargement and cytoarchitectural abnormalities of one cerebral hemisphere. 
; Textual definition: PMID: 22497611  
; PMID to support changes: COST:neuromig 
; label:  
; Curator:  
; OMIM -disease: 605284 191092 601231  614191 611223 171834 601728  
; OMIM -gene

**Comment by pnrobinson** [comment]
This term already exists
https://hpo.jax.org/app/browse/term/HP:0007206
I will add the annotation requests to the HPO annot repo
